.kitchen.docker.yml

```diff
@@ -2,7 +2,7 @@
 ---
 driver:
   name: dokken
-  chef_version: 17.9.26
+  chef_version: 18.3.0
```

Comment:
Why this particular version? It's higher than the version we currently run (17.9.46), so even if it the tests pass, the cookbook could be incompatible with our chef clients.

If you need to bump this, I would prefer 17.9.46 which is the version we run, but then please also bump the version in metadata.rb to have this version as the minimal required version.

Reply:
Updated to ruby 3.0.3 + chef 17.9.26